iOS/PingLocationsViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  PingLocationsViewController.swift
+//  Internet Map
+//
+//  Created by Robert MacEachern on 2019-02-19.
+//  Copyright © 2019 Peer1. All rights reserved.
+//
+
+import UIKit
+
+@objc protocol PingLocationsDelegate {
+    func pingLocationsViewController(_ pingLocationsViewController: PingLocationsViewController, selectedHostName hostName: String)
+}
+
+class PingLocationsViewController: UIViewController {
+
+    @objc var delegate: PingLocationsDelegate?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let backgroundImageView = UIImageView(image: Theme.settingsItemBackgroundImage())
+        view.addSubview(backgroundImageView)
+        backgroundImageView.translatesAutoresizingMaskIntoConstraints = false
+        backgroundImageView.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+        backgroundImageView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+        backgroundImageView.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
+        backgroundImageView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
```

Comment:
Not sure how important it is, but it looks like there are at least  3 places (maybe more not included in this change) that we do this "turn of autoresizing mask and then pin to the same edges of the superview" thing. Might be worth having a utility function or category to do this as a one liner? 

Not sure if it's worth it though, 3 copy-and-pastes is right no the edge of where I'd say it's a no-brainer to have a utility function, so if that's all there is maybe it's not actually worth it. 

Other option would be to wait until NEXT time we have to do something like this, and just add SnapKit.

Reply:
Yeah I would like to bring in Snapkit but I figured we should sort out all the other out of date dependencies (#596) before bringing in a new one. If you feel strongly that this boilerplate-y autolayout code should be improved now, then I can bring it in sooner. Developing our own helpers would be kind of fun but I feel like I'd just end up reimplementing Snapkit's nice interface:
```
box.snp.makeConstraints { (make) -> Void in
    make.edges.equalTo(superview)
}
```
😍